Sources/SwiftInspectorVisitors/FunctionDeclarationVisitor.swift

```diff
@@ -14,16 +17,41 @@ public final class FunctionDeclarationVisitor: SyntaxVisitor {
 }
 
 fileprivate final class FunctionSignatureVisitor: SyntaxVisitor {
+  override func visit(_ node: FunctionParameterSyntax) -> SyntaxVisitorContinueKind {
+    guard
+      let externalLabel = node.firstName?.text,
+      let type = node.type?.typeDescription
+    else { return .skipChildren }
+
+    appendArgument(externalLabel: externalLabel, type: type)
+    return .skipChildren
+  }
   override func visit(_ node: ReturnClauseSyntax) -> SyntaxVisitorContinueKind {
     returnType = node.returnType.typeDescription
     return .skipChildren
   }
 
+  fileprivate func appendArgument(externalLabel: String, type: TypeDescription) {
+    var arguments = self.arguments ?? []
+    arguments.append(.init(externalLabel: externalLabel, type: type))
+    self.arguments = arguments
+  }
+
+  fileprivate var arguments: [FunctionDeclarationInfo.ArgumentInfo]?
   fileprivate var returnType: TypeDescription?
 }
 
 public struct FunctionDeclarationInfo: Codable, Hashable {
   public let name: String
+  public let arguments: [ArgumentInfo]?
   public let returnType: TypeDescription?
-  // TODO: Add parameters and argument labels
+
+  public var selectorName: String {
+    "\(name)(\((arguments ?? []).map { "\($0.externalLabel):" }.joined()))"
+  }
+
+  public struct ArgumentInfo: Codable, Hashable {
+    public let externalLabel: String
```

Comment:
```suggestion
    public let argumentLabelName: String
```

See https://docs.swift.org/swift-book/LanguageGuide/Functions.html#ID166

Reply:
ooh dig that

Reply:
54f69b4